src/test/java/reposense/util/TestUtil.java

```diff
@@ -141,6 +142,29 @@ public static boolean compareNumberExpectedCommitsToGitLogLines(int expectedNumb
         return expectedNumberCommits * 3 - 1 == gitLogResult.split("\n").length;
     }
 
+    /**
+     * Returns true if the {@code expectedNumberFilesChanged} is equal to the actual number of files changed in
+     * {@code gitLogResult}
+     */
+    public static boolean compareNumberFilesChanged(int expectedNumberFilesChanged, String gitLogResult) {
+        // if git log result is empty, then there are no files changed
+        if (gitLogResult.isEmpty()) {
+            return expectedNumberFilesChanged == 0;
+        }
+        String[] changesLogged = gitLogResult.split("\n");
+        HashSet<String> filesChanged = new HashSet<>();
+        // Checks the 2nd line of each commit which contains info of the changed file
+        for (int i = 1; i < changesLogged.length; i += 4) {
+            String log = changesLogged[i];
```

Comment:
Can we start `int i = 0`, and get the `String log = changesLogged[i + 1]`, easier for us to know that we are getting the 2nd line in each iteration.